### PR TITLE
Feature/UI 791

### PR DESCRIPTION
**Pull request by bmorris-revacomm**
- add views for adding 791 data
- if mds starts with "K" show nav link for 791 - if "K" is removed, link is hidden
- added 791 defaults
- remove unnecessary padding, stacks, properties
- change toggle animation because it is conflicting with parent animation
- refactor SortieTypeView to be one view instead of two, clean up code

Note: Somehow John's commits for his PR made their way onto this one? lol
Note 2: Sorry for all the commits lol - I think some weird stuff happened when pulling/pushing and I needed to re-commit some stuff

### CODE REVIEW

File: PBLogging/Views/SingleSortie/Form791/IssuesListView.swift [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+        VStack(alignment: .leading) {
+            HStack {
+                Group {
+                    Text("AIRCRAFT\nCOMMAND")
+                    Text("AIRCRAFT\nTYPE")
+                    Text("AIRCRAFT\nNUMBER")
+                    Text("AIRCRAFT\nCALL SIGN")
+                    Text("AIRCRAFT")
+                    Text("QUANTITY\nISSUED (lbs)")
+                }
```
Comment (mcgarrys) [inline PBLogging/Views/SingleSortie/Form791/IssuesListView.swift:47]: Maybe find a way to line titles up with their textfield text? 

IMG_0038

File: PBLogging/Views/SingleSortie/Form791/IssuesListView.swift [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+                Text("Issues")
+                    .fontSectionHeading()
+                    .padding(.leading)
+                Spacer()
+                ContentButton(isTransparent: true, imageName: "plus", title: "Add Issue") {
+                    addIssue()
+                }
+                .accessibility(identifier: "addIssueButton")
+                .padding(.trailing)
+            }
```
Comment (mcgarrys) [inline PBLogging/Views/SingleSortie/Form791/IssuesListView.swift:31]: Super nitpicky, but could extract that HStack into it's own var like issueListHeader or something. More of a readability thing than anything else.

**Review by mcgarrys — APPROVED** [review]
Overall looks great and works 🔥. Just a few small things/suggestions.

File: PBLogging/Views/SingleSortie/Form791/IssuesListView.swift (hunk 2 above)
Comment (bmorris-revacomm) [reply PBLogging/Views/SingleSortie/Form791/IssuesListView.swift:31]: Oooo good catch

File: PBLogging/Views/SingleSortie/Form791/IssuesListView.swift (hunk 1 above)
Comment (bmorris-revacomm) [reply PBLogging/Views/SingleSortie/Form791/IssuesListView.swift:47]: I just realized this and trying to work on it lol. It's not playing nice :(

File: PBLogging/Views/SingleSortie/Form791/IssuesListView.swift [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+        
+    // MARK: - Initialization
+
+    init(issue: Issue) {
+        func createStateStringFromNumber<T: Numeric>(value: T) -> State<String> {
+            State(wrappedValue: value == 0 ? "" : "\(value)")
+        }
+        
+        _issue = ObservedObject(wrappedValue: issue)
+        _quantityIssuedPounds = createStateStringFromNumber(value: issue.quantityIssuedPounds)
```
[the remark this answers is not in the document]
Comment (bmorris-revacomm) [reply PBLogging/Views/SingleSortie/Form791/IssuesListView.swift:97]: 😬

File: PBLogging/Views/SingleSortie/Form791/IssuesListView.swift (hunk 1 above)
Comment (mcgarrys) [reply PBLogging/Views/SingleSortie/Form791/IssuesListView.swift:47]: I just did something super similar in FlightTimeView, had to use some specific custom padding.

Screen Shot 2021-07-16 at 4 30 53 PM

**Pull request merged**